packages/@okta/vuepress-theme-prose/assets/css/abstracts/_breakpoints.scss

```diff
@@ -1,7 +1,7 @@
 $breakpoints: (
-  'phone': 320px,
-  'tablet': 768px,
-  'desktop': 1024px,
+  'phone': 375px,
+  'tablet': 890px,
+  'desktop': 1250px,
```

Comment:
Why are breakpoints changing?

Reply:
Header is too big, it looks good only till 1250px, after it I resize it. For consistency I changed all breakpoints to a bit larger. What would you suggest?

Reply:
I'll defer to the designers, but I don't think we're expecting breakpoint changes. If there's a design issue with the header, please flag that for the designers.